zstacklib/zstacklib/utils/lvm.py

```
@@ -2743,3 +2750,10 @@ def check_lvmlockd_log():
         except Exception as e:
             logger.warn(str(e))
             self.failed = True
+
+@bash.in_bash
+def stop_lock_service():
```

Ruff (0.14.10)

Redefinition of unused `stop_lock_service` from line 863

(F811)